### ISSUE TITLE
IF YOU HAVE ISSUES WITH DUPLICATE LINES ADDED IN PRS, READ HERE

### ISSUE THREAD

**Issue by phantomic12** [Olivr/free-domain#1044]
A ton of people, including myself at some point had a ton of trouble due to issues relating to duplicate lines in pull requests, but i found a solution and I think it would help a lot of people if I share it here.

if you have a preexisting fork with a failed PR, delete it.

Once you have a clean fork with no modifications, do the following.

open the repo in the github.dev web editor, simply change github.com to github.dev and it should open

Add your website and dns info, but instead of making your line at the bottom of the page, make it between two existing lines

Commit your changes and open a pr, and your info should be sucessfully added!

hope this helps, if anyone needs any help, feel free to comment here and i will be happy to help, as the devs dont seem to care too much, cant really complain though, they are doing a great service to the community as it is

**Comment by forcash2019** [comment]
Hello 
Why I can see my record only when open https://github.dev/forcash2019/free-domain/ and not in https://github.com/Olivr/free-domain/blob/main/zones/wip.la.yaml ?

**Comment by IncognitoTGT** [comment]
When I try to edit name servers on GitHub.dev, the PR still says the newline error

**Comment by phantomic12** [comment]
In my experience changing dns has always been weird, imo ur best bet is to flat out create a new domain name

**Comment by BrMatheusLopes** [comment]
Thank you very much 👍

**Comment by Polkastop** [comment]
ive tried numerous times to get this right, but every time it says more than 1 line changed.

Ive tried it in the beginning, the end , the middel etc.

Ive deleted the fork twice but still doing something wrong.

It doesnt help im a complete newb, all im trying is to add 1 line

poli99: : [ns1.afraid.org, ns2.afraid.org, ns3.afraid.org, ns4.afraid.org] # Register poli99.wip.la

but fail every time, any further advice would be greatly apprciated

thanks ian

**Comment by phantomic12** [comment]
I've actually tried to register another domain to no avail, but imo ur best bet is to try the top/bottom, otherwise i suggest looking for an alternate free domain repo, as im not sure if this one is even maintained anymore

**Comment by santydesignscr** [comment]
I created a website to auto register domains: test.bytewave.wip.la (is beta, but working)

**Comment by santydesignscr** [comment]
My script adds the new domain on the penultimate line since it is the only way to not affect multiple lines.

**Comment by santydesignscr** [comment]
@Polkastop try my website, no email validate yet, use any random

**Comment by thetechnikfreak** [comment]
@santydesignscr Please and and a option to edit the record to your service please

**Comment by santydesignscr** [comment]
It is not possible because the repository does not allow modifying already created domains, that website is only current as one more of us when creating a subdomain, since I do not have any type of special writing access, I also tried to send an email to the author of this subdomain to integrate it directly with the subdomain and replace the repository but I got no response

**Comment by thetechnikfreak** [comment]
@santydesignscr is there a option to delete a record ?

**Comment by santydesignscr** [comment]
No, and it's for the same reason that there isn't a option to delete

**Comment by santydesignscr** [comment]
Hello everyone, it seems that this subdomain is not currently working, I recommend this platform that I just launched, the subdomain is webcr.top, apparently there are almost no subdomains, in my plataform you can register a subdomain, edit NS and remove domains:
https://nic.webcr.top/app
